docs/plugins.md

```diff
@@ -21,6 +21,10 @@ Helm plugins have the following features:
 - They can be written in any programming language.
 - They integrate with Helm, and will show up in `helm help` and other places.
 
+## Plugins Backends
+
+### Native
```

Comment:
Instead of calling them Native and External it may be better to talk about this by system location. System wide and local to a user. This is more akin to what other package managers have done so the concept may be more translatable.